### PR TITLE
Legacy notifications cleanup

### PR DESCRIPTION
**Pull request by tonisevener**
**Phabricator:** 
https://phabricator.wikimedia.org/T302294

### Notes
This is a final cleanup sweep to remove any unused notifications code from the old "in the news" and "edit reverted" local UNNotifications.

### Test Steps
1. Be sure unit tests pass, and be sure the "In the news" card still shows up in the Explore feed.

### CODE REVIEW

File: Wikipedia/Code/WMFAlertManager.swift [hunk 1]
```diff
@@ -34,18 +34,6 @@ open class WMFAlertManager: NSObject, RMessageProtocol, Themeable {
...
-    
-    
-    @objc func showInTheNewsAlert(_ message: String?, sticky:Bool, dismissPreviousAlerts:Bool, tapCallBack: (() -> Void)?) {
-        
-        if (message ?? "").isEmpty {
-            return
-        }
-        showAlert(dismissPreviousAlerts, alertBlock: { () -> Void in
-            let title = CommonStrings.inTheNewsTitle
-            RMessage.showNotification(in: nil, title: title, subtitle: message, iconImage: UIImage(named:"trending-notification-icon"), type: .normal, customTypeName: nil, duration: sticky ? -1 : 2, callback: tapCallBack, buttonTitle: nil, buttonCallback: nil, at: .top, canBeDismissedByUser: true)
```
Comment (staykids) [inline Wikipedia/Code/WMFAlertManager.swift:46]: If I'm not mistaken, `trending-notification-icon` doesn't appear to be used anywhere else. Perhaps we should remove it from the asset catalog as well.

**Review by staykids — COMMENTED** [review]
The enabled tests pass - so long (🤞🏽), old notifications! One small note below and merge conflict to resolve.

File: Wikipedia/Code/WMFAlertManager.swift (hunk 1 above)
Comment (tonisevener) [reply Wikipedia/Code/WMFAlertManager.swift:38]: @staykids Great catch! This has been removed.

**Comment by tonisevener** [comment]
@staykids This is ready for another review!

**Review by staykids — APPROVED** [review]

**Pull request merged**